spec/spec_helper.rb

```diff
@@ -8,29 +8,22 @@
 require "bundler"
 Bundler.setup
 
-ENV['RAILS'] = '4.1.9'
-
 ENV['RAILS_ENV'] = 'test'
+# Ensure the Active Admin load path is happy
+require 'rails'
+ENV['RAILS'] = Rails.version
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.